src/chart/generateCategoricalChart.tsx

```diff
@@ -1103,6 +1104,69 @@ export const generateCategoricalChart = ({
         mouseHandlerCallback: this.triggeredAfterMouseMove,
         layout: this.props.layout,
       });
+      this.displayDefaultTooltip();
+    }
+
+    displayDefaultTooltip() {
+      const { defaultIndex } = this.props;
+      if (
+        typeof this.props.defaultIndex !== 'number' ||
+        defaultIndex < 0 ||
+        defaultIndex > this.state.tooltipTicks.length
+      ) {
+        return;
+      }
+
+      const tooltipElem = (Array.isArray(this.props.children) ? this.props.children : [this.props.children]).find(
+        (child: any) => child.type.name === 'Tooltip',
+      );
```

Comment:
There is a helper function `findChildByType` that could be used here, but also we can hopefully get rid of this soon and instead use context..